src/beta/black-highlighter-beta.css

```diff
@@ -0,0 +1,11730 @@
+/* Korean Language Support */ 
+@import url("https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&display=swap");
+/* Thai Language Support */ 
+@import url("https://fonts.googleapis.com/css2?family=Kanit:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+/* International Monospace Support */ 
+@import url("https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap");
+
+@supports(display: grid) {
+
+/* ROOT VARS
+   ============================= */
+
+/* Structural elements generally part of standard HTML spec as well as Wikidot's base structural elements */
+
+/* ===DIRECTORY===
+	1. =S-CSS-P INTEGRATION
+	2. =HEADER ELEMENTS
+	3. =TYPEFACES
+		3.1 =BASE FONT SIZE & LINE HEIGHT
+		3.2 =HEADER TITLE FONT SIZES
+	4. =VARIABLE FONT SETTINGS
+		4.1 =VARIABLE MONO FONT SETTINGS
+	5. =OPENTYPE FEATURE SETTINGS
+	6. =STANDARD THEME COLORS
+	7. =PRIMARY COLORS
+		8.1. =GENERAL COLORS
+		8.2. =GENERAL TEXT COLORS
+	8. =BACKGROUND & HEADER COLORS
+	9. =MENU COLORS
+		9.1 =MENU BACKGROUND COLORS
+		9.2 =MENU TEXT COLORS
+	10. =SECONDARY & TERTIARY COLORS
+	11. =PRIMARY GRADIENTS
+	12. =STRUCTURAL SPACING ELEMENTS
+		14.1 =HEADER MEASUREMENTS
+		14.2 =TOPBAR MEASUREMENTS
+		14.3 =FINAL HEADER + TOPBAR MEASUREMENTS
+		14.4 =SIDEBAR MEASUREMENTS
+
+	===============================
+	======= SPECIFIC COLORS =======
+	===============================
+	13. =BACKGROUND GRADIENT
+		13.1 =GENERAL BACKGROUND
+		13.2 =HEADER GRADIENT
+		13.3 =TOPBAR GRADIENT
+	14. =LINK COLORS
+	15. =HEADER ELEMENTS
+		15.1 =HEADER TITLES
+		15.2 =SEARCH ELEMENT
+		15.3 =LOGIN BOX ELEMENT
+	16. =TOPBAR ELEMENTS
+		16.1 =TOPBAR CATEGORIES
+		16.2 =DROPDOWN MENU
+	17. =SIDE-BAR ELEMENTS
+		17.1 =SIDE-BAR GENERAL
+		17.2 =SOCIAL MEDIA ICONS
+		17.3 =SIDEBLOCK MENU
+		17.4 =MENU-ITEMS
+		17.5 =SIDEBAR COLLAPSIBLES
+	18. =FOOTER
+	19. =LICENSE AREA
+
+	20. =MAIN CONTENT UI ELEMENTS		
+		20.1 =UI ICONS
+			20.1.1 =PAGE OPTIONS ICONS
+			20.1.2 =TEXT EDITOR ICONS
+		20.2 =TABS
+		20.3 =TABLES
+		20.4 =BLOCKQUOTES
+		20.5 =FOOTNOTES/BIBLIOGRAPHY FOOTER
+		20.6 =TOC
+	
+	21. =GLOBAL ELEMENTS
+		21.1 =UI BUTTONS
+		21.2 =RATING MODULE
+		21.3 =MODALS
+		21.4 =FOOTNOTES HOVER BLOCK
+
+	===============================
+	========= DEPRECIATED =========
+	===============================
+	22. =LEGACY VARS
+  =============== */
+
+:root {
+/* ===S-CSS-P INTEGRATION=== */
+	/* If you're making a new CSS theme, please include the following three variables at minimum. */
+	--theme-base: "black-highlighter";
+	/* must be either "black-highlighter" or "sigma9" */
+	--theme-id: "black-highlighter";
+	/* set this to the URL of your theme's page - eg for "component:ar-theme", set it to "ar-theme" */
+	--theme-name: "nu-SCP Theme";
```

Comment:
This is the theme name, even now?

Reply:
Oh no... we should nuke that. I think the idea initially was that the "Framework" is Black Highlighter but the "Theme" was nu-SCP.

Regardless, nu-SCP can get officially yeeted (even if we are still stuck with github.com/Nu-SCPTheme...

Reply:
Yeah we're stuck with the latter but mostly because we lack a lot of choice here. If we could easily change it I think we would.

Reply:
Definitely would. Alas, we suffer from the folly of our youth...